src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.Management.tests.ps1

```diff
@@ -7,7 +7,7 @@ InModuleScope Arcus.Scripting.Management {
             $config = & $PSScriptRoot\Load-JsonAppsettings.ps1
             & $PSScriptRoot\Connect-AzAccountFromConfig.ps1 -config $config
         }
-        Context "Remove soft deleted Azure API Management service" {
+        Context "Remove soft deleted Azure API Management service" -Skip {
```

Comment:
It fails with 503 ServiceUnavailable, but I'm wondering if in this case we do not have the permissions for our SerivcePrincipal in the new Dev tenant; but I could be wrong. Not sure what we need this test and/or we can even test it in the new Dev tenant.

Reply:
Just to be sure, could you give this test another go? I read something that this might be related to an Azure outage, changes are small but cannot hurt to give this another test

Reply:
Testing...

Reply:
I'm actually wondering why we have that in the first place, as you can specify `latest` I believe.
I'll remove those and see.

Reply:
@pim-simons , that worked! You think it's fine now?

Reply:
Looks good to me!